delfin/exporter/base_exporter.py

```diff
@@ -101,11 +102,25 @@
     def __init__(self):
         super(PerformanceExporterManager, self).__init__(self.NAMESPACE)
 
+    def exporter_enable(self, exporter):
+        try:
+            if os.environ[exporter] == "True":
+                return True
+            return False
+        except KeyError:
+            os.environ[exporter] = "False"
+
     def dispatch(self, ctxt, data):
         # create object of prometheus class and push to prometheus
         # exporter to translate into time-series format
-        prometheus_obj = prometheus.PrometheusExporter()
-        prometheus_obj.push_to_prometheus(data)
+        if self.exporter_enable("PROMETHEUS"):
```

Comment:
This should be in some constant file or named better like DELFIN_ENABLE_PROMETHEUS
But as this logic need to be improved to read from conf file in future so its fine. Just open an issue

Reply:
Opened an issue to capture all issues identified to improve user experience .
https://github.com/sodafoundation/delfin/issues/350